gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystem.java

```diff
@@ -63,14 +63,16 @@ public GoogleHadoopFileSystem() {
    * GoogleCloudStorageFileSystem; initialize() will not re-initialize it.
    */
   @VisibleForTesting
-  GoogleHadoopFileSystem(GoogleCloudStorageFileSystem gcsfs) {
+  //Bhagyaa-s
+  public GoogleHadoopFileSystem(GoogleCloudStorageFileSystem gcsfs) {
     super(gcsfs);
   }
-
+//Bhagyaa-e
   /** Sets and validates the root bucket. */
   @Override
   @VisibleForTesting
-  protected void configureBuckets(GoogleCloudStorageFileSystem gcsFs) throws IOException {
+  //Bhagyaa-s Bhagyaa-e
+  public void configureBuckets(GoogleCloudStorageFileSystem gcsFs) throws IOException {
```

Comment:
Do we need this method to be public ?

Reply:
No, we do not. I used that to use TrackingHttpRequests as a file outside of tests. I will revert these changes